appengine/flexible/django_cloudsql/mysite/settings.py

```diff
@@ -79,15 +79,15 @@
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'polls',
-        'USER': 'root',
-        'PASSWORD': '<your-root-password>',
+        'USER': '<your-database-user>',
+        'PASSWORD': '<your-database-password>',
     }
 }
 # In the flexible environment, you connect to CloudSQL using a unix socket.
 # Locally, you can use the CloudSQL proxy to proxy a localhost connection
 # to the instance
 DATABASES['default']['HOST'] = '/cloudsql/<your-cloudsql-connection-string>'
-if os.getenv('GAE_APPENGINE_HOSTNAME'):
+if os.getenv('GAE_INSTANCE'):
```

Comment:
I think this change only makes sense if you switch it to env: flex, which we haven't done yet?